### PR TITLE
Home page frontend

### LINKED ISSUE #44: Home Page Front-End
Home Page which is a landing page where you set the layout which will contain  related to issue #8

1. NavBar with the logo 

1. Image for the landing page 

1. Paragraph for a description of our software

1. Google Loin Button

### PR DESCRIPTION
**Pull request by karmelyoei**
### What does this PR do?
- Displaying the home page of our website Frontend

### Does this Fix an Issue?
Fixes #44

### How to test this PR
- npm i 
- cd client && npm i
- npm run dev

### Screenshots of changes

Landing Page

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -1,20 +1,35 @@
...
 
 function App() {
 	return (
 		<Router>
-			<Switch>
-				<Route exact path="/dashboard" component={NewMember} />
-				<Route path="/" exact component={Home} />
-				{/* <Route path="/dashboard" exact component={Dashboard} /> */}
-			</Switch>
+			<Grid direction="column" container>
```
Comment (yaqootturman) [inline client/src/App.js]: Why Grid in the router!!!!

File: client/src/components/CommonComponents/SignIn/index.jsx [hunk 2]
```diff
@@ -4,6 +4,7 @@ import 'firebase/auth';
 import 'firebase/firestore';
 import StyledFirebaseAuth from 'react-firebaseui/StyledFirebaseAuth';
 import uiConfig from './uiConfig';
+import styles from './style'; // This uses CSS modules.
```
Comment (yaqootturman) [inline client/src/components/CommonComponents/SignIn/index.jsx]: You imported style but you didn`t use/call it, why?

File: client/src/components/LayoutComponents/Home/index.jsx [hunk 3]
```diff
@@ -1,20 +1,44 @@
...
-			</StylesProvider>
-		</div>
-	);
+class Home extends Component {
+	render() {
+		const { classes } = this.props;
+		return (
+			<Box width={1} mt={4}>
+				<Paper elevation={3} className={classes.paper}>
+					<Grid container xs={12} justify="flex-start" alignItems="center">
```
Comment (yaqootturman) [inline client/src/components/LayoutComponents/Home/index.jsx]: I don't see the useful point about using Grid here?

File: client/src/components/LayoutComponents/Home/style.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+const Style = {
+	paper: {
+		background: '#E4DDDD',
+		width: 1170,
```
Comment (yaqootturman) [inline client/src/components/LayoutComponents/Home/style.js:4]: The style not responsive and it's large to screen in desktop one!

**Review by yaqootturman — CHANGES_REQUESTED** [review]

File: client/src/App.js (hunk 1 above)
Comment (karmelyoei) [reply client/src/App.js]: its a standard template for all of us to make sure we are having the same space on the page

File: client/src/components/CommonComponents/SignIn/index.jsx (hunk 2 above)
Comment (karmelyoei) [reply client/src/components/CommonComponents/SignIn/index.jsx]: yes cause i wanna try to find a way to change the Google sign-in design according to our design

File: client/src/components/LayoutComponents/Home/index.jsx (hunk 3 above)
Comment (karmelyoei) [reply client/src/components/LayoutComponents/Home/index.jsx]: its useful especially forme when i want to do it responsible make it easier

File: client/src/components/LayoutComponents/Home/style.js (hunk 4 above)
Comment (karmelyoei) [reply client/src/components/LayoutComponents/Home/style.js:4]: really on mine its not but you are right about responsive I will do it today

**Review by mantagen — DISMISSED** [review]
LGTM :)

**Comment by yaqootturman** [comment]
It shows me a blank page! 
Add the temple, Please!

**Review by yaqootturman — DISMISSED** [review]

**Review by rubasider — APPROVED** [review]

**Review by yaqootturman — APPROVED** [review]

**Pull request merged**